lib/esbuild/css_printer/css_printer.go

```diff
@@ -37,6 +37,7 @@ type Options struct {
 	AddSourceMappings bool
 	LegalComments     config.LegalComments
 	Scope             string
+	ScopeStrategy     int
```

Comment:
Could also update this type to use the enum

```suggestion
	ScopeStrategy     ScopeStrategy
```